code/core/src/manager/container/Sidebar.tsx

```diff
@@ -30,20 +30,11 @@ const Sidebar = React.memo(function Sideber({ onMenuClick }: SidebarProps) {
       refs,
     } = state;
 
-    const menu = useMenu(
-      state,
-      api,
-      showToolbar,
-      api.getIsFullscreen(),
-      api.getIsPanelShown(),
-      api.getIsNavShown(),
-      enableShortcuts
-    );
```

Comment:
I'm pretty sure this was not an object, and a flat list of arguments for cheap memoization; converting it to an singular object re-created every render defeats this.